src/Seo/AttributeBag.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\SeoBundle\Seo;
+
+final class AttributeBag implements \IteratorAggregate
+{
+    private $attributes = [];
```

Comment:
You could add a PHPDoc here:
```suggestion
   /**
     * @param array<string|string>
     */
    private $attributes = [];
```